docs/tutorials/typescript/work-queue-slack-app/index.mdx

```diff
@@ -0,0 +1,991 @@
+
+### Use case
+
+**What's the application you are building a what's the use case?**
+
+You are building a Work Queue Slack App.
+
+Imagine an organization with dozens and dozens of teams.
+Each team provides a service to the organization, and each team has a Slack channel.
+While there may be a common task tracking system in place, there are often many micro tasks that don't warrant a full task in the task tracking system.
+For example, an engineer may need a review on a Pull Request from another team they're integrating services with.
+Often these micro task requests come through Slack conversations.
+Each team might have a dedicated on-call person responsible for handling these requests.
+On a busy day, it could be possible to lose track of all the requests in the Slack channel.
+It would be nice to have a standard way for folks to submit requests and for the on-call person to manage them.
+
+The application you are building today is a Slash command Slack App to submit and manage general "work" requests in a Slack channel.
+
+### System architecture
+
+**What are the components of the system and how are they arranged?**
+
+The system consists of a Slack Workspace, a Slack App, a Temporal Service, and a Temporal Application.
+
+Traditionally, you would need a database to persist the state of your Slack App between interactions in case the Slack App process crashes or goes down.
+If you use Temporal, you can persist the state of your Slack App directly within a Temporal Application.
+
+![System component architecture](./images/temporal-slack-app.svg)
+
+The preceding diagram abstracts Slack's back end as a communication channel between the Slack App and the Slack Workspace.
+The diagram could also abstract the Temporal Service as a communication channel.
+However, for folks learning Temporal, it's helpful to understand that the Temporal Service is a separate service from the Temporal Application.
+
+**Is trading off a database for Temporal worth it?**
+
+Setting up and managing your database can be more overhead.
+During the development process instead of a managing a database, you will run a Temporal Service locally alongside your Temporal Application.
+You should already have it running if you completed the [Prerequisite steps](#prerequisites).
+So, during development it will seem like you're managing two things instead of one.
+
+However, after the development process, if you use Temporal Cloud as your Temporal Service, you only need to deploy the Temporal Application the same way as you would your Slack App.
+Then, your Temporal Application replaces an entire database with less than 70 lines of code.
+
+![Temporal development vs production](./images/temporal-slack-app-dev-vs-prod.svg)
+
+To use Temporal Cloud in production, see the [Deploy your TypeScript application to Digital Ocean](/tutorials/typescript/digital-ocean-deployment) tutorial for more information.
```

Comment:
Don't think we need this here, it's already in the conclusion (good work on that)